warmups/rotation/README.md

```diff
@@ -0,0 +1,15 @@
+# Create a React App!
+
+In this warmup the goal is to practice making a React app from the ground up! 
+
+You have been tasked with delivering a React app to the client using their given business logic (found in  `rotation.js`).
+
+The wireframe from the product owner for this project is [here](https://wireframe.cc/KePV8d).
+
+If you have any questions, feel free to ask!
+
+A few guidelines to remember:
+- You will want a linter
+  - This client hates semi-colons
+- You will want some tests
+- The client is color blind, so any color scheme you may or may not choose will be awesome!
```

Comment:
"color-blindness" is a legitimate disability and as such, when we take that into account, just the opposite is true: certain color schemes are definitely not awesome! 😉

Reply:
That is a very good point. Oops!